### ISSUE TITLE
Transition Nodes in AnimationTree do not take current transition into account

### ISSUE THREAD

**Issue by ghost** [godotengine/godot#7067]
**Operating system or device - Godot version:**
2.1 stable

**Issue description** (what happened, and what was expected):
When using the AnimationTree Transition Node to transition to a new animation, the current transition taking place is discarded. For example, if you use a transition node to swap between Idle/Walk, quickly tapping on the move key will result in the transition snapping to its end before starting a new transition, rather than transitioning from the point it currently is at.

**Steps to reproduce:**
Transition from one animation then back again before the transition is complete.

You can see it taking place with my directional swimming animations in this video. Instead of smoothly transitioning between the four different possible animations it snaps to the end of the transition before starting a new transition.
https://www.youtube.com/watch?v=BaaCedLTpQc

I've already gotten around this limitation for my walk animations using blend nodes and script to transition between them, but it would just be a lot easier if this worked nicely by default.

**Comment by Noshyaar** [comment]
First of all thank you for your report and sorry for the delay.

We released Godot 3.0 in January 2018 after 18 months of work, fixing many old issues either directly, or by obsoleting/replacing the features they were referring to.

We still have hundreds of issues whose relevance/reproducibility needs to be checked against the [current stable version](https://godotengine.org/download), and that's where you can help us.
**Could you check if the issue that you described initially is still relevant/reproducible in Godot 3.0 or any newer version, and comment about it here?**

For bug reports, please also make sure that the issue contains detailed steps to reproduce the bug and, if possible, a zipped project that can be used to reproduce it right away. This greatly speeds up debugging and bugfixing tasks for our contributors.

Our Bugsquad will review this issue more in-depth in 15 days, and potentially close it if its relevance could not be confirmed.

Thanks in advance.

*Note: This message is being copy-pasted to many "stale" issues (90+ days without activity). It might happen that it is not meaningful for this specific issue or appears oblivious of the issue's context, if so please comment to notify the Bugsquad about it.*

**Comment by akien-mga** [comment]
As there was no answer to the above request, we're closing this issue as inactive.
If you can still reproduce it, please open a new issue against a currently supported release of Godot.
